### PR TITLE
Amirhossein/on chain init admin

### PR DESCRIPTION
**Pull request by RadNi**
#249 closed.

### CODE REVIEW

**Comment by RadNi** [comment]
Test for non-admin app initialization.
 #251 closed.

File: mlabs/test/Test/NFT/Script/Values.hs [hunk 1]
```diff
@@ -79,7 +79,7 @@ testStateAddr = NFT.txScrAddress
 
 -- FIXME
 appInstance :: NftAppInstance
```
Comment (cstml) [inline mlabs/test/Test/NFT/Script/Values.hs:81]: Could this be fixed?

**Review by zygomeb — COMMENTED** [review]
@cstml can you double check this one? I'm not sure about the admin parameters here and if we need them at all

**Comment by cstml** [comment]
> @cstml can you double check this one? I'm not sure about the admin parameters here and if we need them at all

It was part of the [spec described previously](https://github.com/mlabs-haskell/plutus-use-cases/blob/vlad/spec/mlabs/docs/NFT/Spec.agda). The purpose was to ensure that we had some super-users which could manage/open/close the app. Going forward with the (simple) Governance variation, we probably can drop the admin list and have some `Vote` condition based on Governance held (though it would need to be described/specced prior), together with a safe way of turning the app from `active` to `not active` (for example when migrating to a different version).

**Pull request merged**